hack/ensure-tools.sh

```diff
@@ -53,4 +53,14 @@ ensure_kustomize() {
     chmod 0755 "${_lcl_bin}"
 }
 
+ensure_envsubst() {
+  echo "ensure-tools: envsubst"
+
+  if [ ! envsubst --help 2>/dev/null ]; then
```

Comment:
Hi @andrewsykim,

A few notes:

* The brackets aren't necessary. Please enable the [shellcheck](https://www.shellcheck.net) plug-in for you IDE.
* Please change to `if ! command -v envsubst >/dev/null 2>&1` to be consistent with the standard way to check the existence of commands.